doc/interface/spatial_discretization_methods.rst

```diff
@@ -50,72 +49,72 @@ Having the exact solution, we can compute an experimental order of convergence (
-- Problems with strong gradients, e.g., no or low dispersion and bindings that create sharp fronts
-- Bindings that mathematically require positive values or exhibit strange behaviour with negative concentration values
+- Problems with strong gradients, e.g., no or low dispersion and binding model parameters that create sharp fronts
+- Binding models that mathematically require positive values or exhibit strange behaviour with negative concentration values
 
 We recommend the DG method for
 
-- Large problem sizes, e.g., high resolutions and more complex models (i.e. the LRMP and specifically the GRM)
-- Smooth problems, e.g., sufficient dispersion
+- Large problem sizes, e.g., high spatial resolutions and more complex models (i.e. the LRMP and specifically the GRM)
+- Smooth problems, i.e., sufficient band broadening
 
 Recommendations on DG discretization parameters
 -----------------------------------------------
 
 - Employ an axial polynomial degree between 3 and 5
-- Select a lower axial polynomial degree for non smooth tendency and employ more elements instead. Converse choice for smooth problems
-- Adjust the DG particle polynomial degree to control approximation accuracy; leave the number of elements at one. Make exceptions if very steep gradients occur inside the particles or when specific parts of the particle domain are more interesting (here, you can resolve more interesting parts by a user-defined spacing of multiple elements)
-- The field ``EXACT_INTEGRATION`` specifies the DG polynomial integration method. The default value of 0 (collocation DG) is expected to be slightly more performant in most settings
+- Select a lower axial polynomial degree for approximating functions that tend to be less smooth and employ more elements instead. Converse choice for smooth problems
+- Adjust the DG particle polynomial degree to control approximation accuracy; leave the number of elements at one. Make exceptions if very steep gradients occur inside the particles or when specific regions of the particle domain are more interesting (the spatial resolution of certain regions can be refined by a user-defined spacing of multiple elements)
+- The field ``EXACT_INTEGRATION`` specifies the DG polynomial integration method. The default value of $0$ (collocation DG) is expected to be slightly more performant in most settings
 
 Refinement strategy
 -------------------
 
 A common problem in numerical simulation is that the number of discrete points required to yield an accurate approximation within a specific tolerance is unknown.
 We thus recommend determining the approximation error via comparison with a refined reference approximation.
 Both the theoretical order of convergence and the EOC can be used to estimate the required number of discrete points.
+An EOC that is significantly lower than the theoretical order indicates that the problem is numerically underresolved.
 
 Note on DG solution vector
```

Comment:
@eric asked to add a description to how thats done since this is not an option supported by cadet itself